x-pack/plugins/enterprise_search/public/applications/app_search/components/documents/search_experience/views/multi_checkbox_facets_view.tsx

```diff
@@ -14,20 +14,16 @@ import {
   EuiButtonEmpty,
   EuiSpacer,
 } from '@elastic/eui';
+import type { FieldValue, FacetValue } from '@elastic/search-ui';
 import { i18n } from '@kbn/i18n';
 
-interface Option {
-  value: string;
-  selected: boolean;
-}
-
 interface Props {
   label: string;
-  options: Option[];
+  options: FacetValue[];
   showMore: boolean;
   onMoreClick(): void;
-  onRemove(id: string): void;
-  onSelect(id: string): void;
+  onRemove(value: FieldValue): void;
+  onSelect(value: FieldValue): void;
```

Comment:
This change matches the Search UI types, but they might be incorrect. In fact, we do type casting to prevent this error in Search UI too: https://github.com/elastic/search-ui/blob/17488c45f21a496ad5cfbe1e09e624240647b93b/packages/react-search-ui-views/src/MultiCheckboxFacet.tsx#L40

Added to issue to investigate: https://github.com/elastic/enterprise-search-team/issues/2226